.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
I do not think we need to add vscode settings to recharts.

Reply:
+1, haven't looked at the rest of the PR yet but editor settings should be in our gitignore. Each devs editor settings are their own and we'll let eslint/prettier take care of formatting